src/compiler/transformers/module/module.ts

```diff
@@ -745,11 +745,20 @@ namespace ts {
                     else if (declaration.kind === SyntaxKind.ImportSpecifier) {
                         const name = (<ImportSpecifier>declaration).propertyName
                             || (<ImportSpecifier>declaration).name;
-                        return createPropertyAccess(
-                            getGeneratedNameForNode(declaration.parent.parent.parent),
-                            getSynthesizedClone(name),
-                            /*location*/ node
-                        );
+                        if (name.originalKeywordKind === SyntaxKind.DefaultKeyword && languageVersion <= ScriptTarget.ES3) {

```

Comment:
This is the brhaviour in master today as well. In the future we should add an es3 transform that will write keywords correctelly as well as trailing commas removal etc.